### PR TITLE
Add new entries to the bibliography and use short references uniformly in the code

### PR DESCRIPTION
**Pull request by pleroy**

### CODE REVIEW

File: documentation/bibliography.bib [hunk 1]
```diff
@@ -965,6 +1196,54 @@ @inproceedings{PellegriniRussel2014
...
+  title       = {TOPEX/Poseidon Orbit Maintenance for First Five Years},
+  type        = {Technical Report},
+}
+
+@report{BlancSchgounn1996,
+  author      = {Blanc, F. and Schgounn, C.},
+  institution = {Centre national d'études spatiales},
+  date        = {1996-07},
+  number      = {AVI-NT-02-101-CN},
+  title       = {AVISO/Altimetry, AVISO User Handbook for Merged TOPEX/POSEIDON products, Edition 3.0},
```
Comment (eggrobin) [inline documentation/bibliography.bib:1244]: This entry has an overfull hbox; add \- in TOPEX/POSEIDON as needed.

File: documentation/bibliography.bib [hunk 2]
```diff
@@ -836,6 +992,31 @@ @book{ByrdFriedman1954
   title     = {Handbook of elliptic integrals for engineers and physicists},
 }
 
+@book{Capderou2012,
+  author    = {Capderou, M.},
+  publisher = {Springer-Verlag France},
+  date      = {2012},
+  isbn      = {978-2-287-99049-6},
+  title     = {Satellites: de Kepler au GPS},
```
Comment (eggrobin) [inline documentation/bibliography.bib:1000]: `Satellites~: de Kepler au GPS`.

File: integrators/embedded_explicit_generalized_runge_kutta_nyström_integrator.hpp [hunk 3]
```diff
@@ -43,18 +43,15 @@ using quantities::Variation;
...
+// a′).  However, note that Fine uses ^ for the low order method, whereas we use
+// it for the high order method, consistently with the notation used by Dormand,
 // المكاوى, and Prince for RKN methods.
 
 // Alternative notations use a and b for the velocity Runge Kutta matrix and
-// weights: Murua (1998), Runge-Kutta-Nyström methods for general second Order
-// ODEs with application to multi-body Systems, uses (c, α, a, β̂, b̂, β, b),
-// and Dormand (1996), Numerical Methods for Differential Equations: A
-// Computational Approach, uses (c, ā, a, b̄̂, b̂, b̄, b).
+// weights: [Mur98] uses (c, α, a, β̂, b̂, β, b), and [Dor96], uses
```
Comment (eggrobin) [inline integrators/embedded_explicit_generalized_runge_kutta_nyström_integrator.hpp:53]: `[Dor96] uses` (remove the comma).

File: integrators/methods.hpp [hunk 4]
```diff
@@ -701,25 +692,13 @@ struct McLachlanAtela1992Order5Optimal : SymplecticRungeKuttaNyström {
                                                    -0.0589796254980311632}}};
 };
 
-// This integrator goes by many names, see Hairer, Lubich, and Wanner (2003),
-// Geometric numerical integration illustrated by the Störmer–Verlet method,
-// http://www.math.kit.edu/ianm3/lehre/geonumint2009s/media/gni_by_stoermer-verlet.pdf,
-// section 1.3 (Historical remarks).
+// This integrator goes by many names, see [HLW03], section 1.3 (Historical
+// remarks).
 // Notably, it appears in Philosophiae Naturalis Principia Mathematica,
```
Comment (eggrobin) [inline integrators/methods.hpp:697]: Should we have [New87]?

File: numerics/elliptic_functions.hpp [hunk 5]
```diff
@@ -2,12 +2,8 @@
 
 #include "quantities/quantities.hpp"
 
-// This code is a derived from: Fukushima, Toshio. (2012). xgscd.txt (Fortran
-// program package to compute the Jacobian elliptic functions, sn(u|m), cn(u|m),
-// dn(u|m)).  Downloaded from:
-// https://www.researchgate.net/publication/233903220_xgscdtxt_Fortran_program_package_to_compute_the_Jacobian_elliptic_functions_snum_cnum_dnum
-// The original code has been translated into C++ and adapted to the needs of
-// this project.
+// This code is a derived from: [Fuk12a].  The original code has been translated
```
Comment (eggrobin) [inline numerics/elliptic_functions.hpp:5]: `is derived`

**Review by eggrobin — APPROVED** [review]

File: integrators/methods.hpp (hunk 4 above)
Comment (pleroy) [reply integrators/methods.hpp:697]: Done, also with a reference in `ephemeris_body.hpp`.  Let's hope that we don't have to reference a paper by Newhall from 1987...

File: documentation/bibliography.bib (hunk 2 above)
Comment (pleroy) [reply documentation/bibliography.bib:1000]: Also did the same for Kahan's mindless question mark.

**Pull request merged**